Dockerfile

```diff
@@ -61,6 +61,8 @@ RUN apt-get install -y software-properties-common \
   && gem update --system 3.0.3 \
   && gem install bundler -v 1.17.3 --no-document
 
+# NOTE: For some reason this isn't succeeding if installed in the above RUN
+RUN gem install bundler -v 2.2.11 --no-document
```

Comment:
Installed and used to `bundle install` core's dependencies but not used in updates.